specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md

```diff
@@ -293,8 +295,6 @@ are listed in the table below:
 | Unpackaged     | Managed  | Read, Write  | Read, Write | Read, Write  |
```

Comment:
Line 286: " We are introducing the new API so that non-fulltrust packaged apps can access some of the same functionality..."

AppContainer isn't just for UWP (and never was).

SUGGESTION: "We are introducing the new API so non-fulltrust apps can access the same functionality...`